src/PythonMigrationViewExtension/Controls/BaseDiffViewer.xaml.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Windows;
+
+namespace Dynamo.PythonMigration.Controls
+{
+    /// <summary>
+    /// Interaction logic for BaseDiffViewer.xaml
+    /// </summary>
+    public partial class BaseDiffViewer : Window
+    {
+        private PythonMigrationAssistantViewModel ViewModel { get; set; }
+
+        internal BaseDiffViewer(PythonMigrationAssistantViewModel viewModel)
+        {
+            ViewModel = viewModel;
+            DataContext = viewModel;
+            InitializeComponent();
+        }
+
+        private void OnDiffButtonClick(object sender, RoutedEventArgs e)
+        {
+            if (ViewModel.CurrentViewModel.ViewMode == Differ.ViewMode.Inline)
+            {
+                ViewModel.ChangeViewModel(Differ.ViewMode.SideBySide);
+                Width = 1200;
+            }
+
+            else
+            {
+                ViewModel.ChangeViewModel(Differ.ViewMode.Inline);
+                Width = 600;
+            }
```

Comment:
As we discussed, if we want the code in the diff viewer to use the exact same wrapping as the code editor, we will need some fine tuning to these sizes. A good test for this is the line `print 'a very long line that will wrap and will allow us to test if the 2 3 Python migration window respects the same spacing as the Python script editor window'`. This line would break after `3` in the code editor but before `2` in the diff viewer initially.

Reply:
Also, the side by side viewers are a little smaller than the inline viewer, probably because we are not accounting the width of the separator. Also something to consider while fine tuning these sizes.

Reply:
fixed now, please see screenshots below.

Reply:
Hi @radumg . This is looking better, but I can still see a slight difference with the example I provided (the `3` appears on the first line in the code editor):
<img width="1483" alt="Screen Shot 2020-07-30 at 8 32 16 AM" src="https://user-images.githubusercontent.com/10048120/88923372-9c94ee00-d23f-11ea-95bf-0d374efaa2ba.png">
However, after changing to inline view, the wrapping matches the expectations and switching back to side by side view now it also looks as expected. I guess there is something we are not accounting for when displaying the initial window but then we are when switching back and forth?

Reply:
hm, that sounds curious as initial display and switching should follow same path and be initialised the same.

Reply:
Just tested and can't replicate it, the `3` is on the next line for me when the script editor opens

![image](https://user-images.githubusercontent.com/11439624/88929157-07631b00-d27a-11ea-81a1-2a22fb0e1cc6.png)

Reply:
could this be a high-DPI or scaling issue? Both my laptop and external monitor are 4K with scaling set to the recommended `150%`

Reply:
That is strange indeed. I wonder why it the script editor shows different for us 🤔 Here is my screenshot:
<img width="1505" alt="Screen Shot 2020-07-30 at 9 38 38 AM" src="https://user-images.githubusercontent.com/10048120/88929814-a2430180-d248-11ea-870f-e83aa59999a2.png">

Reply:
i notice Safari in the background, you running Windows virtualised (Parallels, VmWare) on Mac ? Not sure if it'd make a difference, but it might affect visual fidelity somehow ?

Reply:
there's something going on with the window borders on your screenshot ?
Might be skewing things a few pixels width, maybe enough to bump the `3` to next line ?

![image](https://user-images.githubusercontent.com/11439624/88931291-ee0f9e00-d27c-11ea-96df-6b79a0547901.png)

![image](https://user-images.githubusercontent.com/11439624/88931337-fa93f680-d27c-11ea-9f8b-b2d6580dd118.png)

Reply:
@radumg My settings are different, so I guess it could be. I'm on 250% DPI. The funny thing is that WPF is supposed to isolate us from these problems, but maybe it's like a 1 pixel issue rounding differently for me than for you.

Reply:
@radumg Just checked and yes, with 150% DPI they both match. I don't know if there is something you can do on your end, but at least you have a way to reproduce it I guess. Let us know if you would like to try.

Reply:
I would say it's not related to parallels because I can reproduce it on 250% when running outside coherence too.

Reply:
just tried and 250% produces the same result for me 🤷‍♂️

Also, don't think anyone with a Windows laptop would actually be on 250% scaling unless using an 8K display 😂

![image](https://user-images.githubusercontent.com/11439624/88936224-f79c0480-d282-11ea-840a-c3a64a5c6ed0.png)